packages/benchmark/src/benchmarkerator.js

```diff
@@ -509,16 +576,26 @@ export const makeBenchmarkerator = async () => {
     agoricNamesRemotes,
   );
 
-  const actors = {
-    gov1: await walletFactoryDriver.provideSmartWallet(
+  const governanceDriver = await makeGovernanceDriver(
+    swingsetTestKit,
+    agoricNamesRemotes,
+    walletFactoryDriver,
+    [
       'agoric1ldmtatp24qlllgxmrsjzcpe20fvlkp448zcuce',
-    ),
-    gov2: await walletFactoryDriver.provideSmartWallet(
       'agoric140dmkrz2e42ergjj7gyvejhzmjzurvqeq82ang',
-    ),
-    gov3: await walletFactoryDriver.provideSmartWallet(
       'agoric1w8wktaur4zf8qmmtn3n7x3r0jhsjkjntcm3u6h',
-    ),
+    ],
+  );
+
+  const liquidationTestKit = await makeLiquidationTestKit({
```

Comment:
seems odd for `benchmarkerator` to be concerned with liquidation per se.

consider having a way for each benchmark to load up tools it needs.

Reply:
Benchmarkerator is trying to establish a complete set of everything a vat might encounter on chain, and, as I discovered in the course of implementing the new benchmarks here, the liquidation machinery is part of that.  I really would prefer better modularity for stuff like this, but that would require an additional layer of refactoring of the support tooling in `boot` that I don't think pencils out in terms of ROI.